sass/utils/_mixins.scss

```diff
@@ -0,0 +1,37 @@
+// MEDIA QUERY 
+/*
+0 - 600px:      Phone
+600 - 900px:    Tablet portrait
+900 - 1200px:   Tablet landscape
+[1200 - 1800] is where our normal styles apply
+1800px + :      Big desktop
+
+$breakpoint arguement choices:
+- phone
+- tab-port
+- tab-land
+- big-desktop
+
+ORDER: Base + typography > general layout + grid > page layout > components
+
+1em = 16px
+*/
+@mixin respond($breakpoint) {
+
+    @if $breakpoint == phone-small {
+        @media only screen and (max-width: 22.5em) { @content };    //360px
```

Comment:
Czy ja się mylę, czy to niezbyt zadziała? Będziesz miał ustawienia responsywności zależne od rozmiaru fontów, a powinny być zależne od rozmiarów ekranu. Może dlatego masz bardzo małe literki :) ? 

![image](https://user-images.githubusercontent.com/18637116/102724256-a3b3ba00-430e-11eb-97e4-ecdc6ee8083f.png)